stock_picking_report_product_sticker/views/stock_picking_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='other_infos']" position="inside">
+                <field
+                    name="show_product_stickers"
+                    readonly="0"
```

Comment:
Please set it as `readonly=False` at the model level instead (see comment above):

```suggestion
```